soc/arm/microchip_mec/mec172x/reg/mec172x_gpio.h

```diff
@@ -204,7 +213,8 @@ enum mec_gpio_idx {
 	MCHP_GPIO_0035_ID,
 	MCHP_GPIO_0036_ID,
 	MCHP_GPIO_0040_ID = 32,
-	MCHP_GPIO_0042_ID = 34,
+	MCHP_GPIO_0041_ID,
+	MCHP_GPIO_0042_ID,
```

Comment:
Any specific reason for the exclusion of below member?
I see those bits are set in BITMAP definition.

MCHP_GPIO_0001_ID
MCHP_GPIO_0005_ID
MCHP_GPIO_0006_ID
MCHP_GPIO_0230_ID to MCHP_GPIO_0236_ID
MCHP_GPIO_0251_ID and MCHP_GPIO_0252_ID